packages/cli/src/cmds/preview/teamsAppInstallation.ts

```diff
@@ -125,8 +126,13 @@ export async function showInstallAppInTeamsMessage(
 
 export async function getTeamsAppInternalId(appId: string): Promise<string | undefined> {
   // TODO: handle GraphTokenProviderUserPassword
```

Comment:
Is this `TODO` still valid?

Reply:
Talked with kuojian, this todo job is no longer needed